### ISSUE TITLE
Viomi V2 (viomi.vacuum.v6) verified working!

### ISSUE THREAD

**Issue by BrendanxP** [nergal/homeassistant-vacuum-viomi#31]
I own a chinese Viomi V2 (viomi app) which has been re-flashed with German/European Xiaomi software (Mi Home App) by Viomi itself.
Thus, it registers as a viomi.vacuum.v6 and it seems to work great with your integration! If you need any logs/proof please send me a message, otherwise you can close this issue and mark it as verified :)

**Comment by rob1303-hub** [comment]
Agreed - I have 2x Australian Vuimi models - not reflashed at all - and they work without issues,
Thank you.
